odkx-src/suitcase-intro.rst

```diff
@@ -57,26 +59,40 @@ ODK-X :guilabel:`Cloud Endpoint Address`, the :guilabel:`App ID`, and your :guil
 Otherwise, please specify an ODK-X Cloud Endpoint username and password with sufficient permissions.
 
   .. note:: 
-    By default :guilabel:`App ID = default`, check the second paragraph of :ref:`ODK-X Data Management Applications <build-app-data-mgt-apps>` to see more details.
+    By default :guilabel:`App ID = default`,to use your own App ID uncheck the option to use default App ID, check the second paragraph of :ref:`ODK-X Data Management Applications <build-app-data-mgt-apps>` to see more details.
 
-There are two tabs at the top of the graphical interface:
-  - :guilabel:`Download`, to download existing data from the server
-  - :guilabel:`Upload`, to upload new data to the server. The :guilabel:`Upload` tab also includes an option to :guilabel:`Reset` the server.
+Suitcase remembers the last username and server url entered. You can check the :guilabel:`save password` option to keep yourself logged in when you open the application again. To log out of the application you can use the logout button in a menu on the top left of the Application.
+
```

Comment:
Please add a screenshot of the logout button I couldn't see it in the screenshots. Also application can be lower case